new/LIT7035Tasabha.xml

```diff
@@ -0,0 +1,80 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ተሰብሐ፡ ሰማዕተ፡ ልዳ፡ በልሳነ፡ ሰይፍ፡ ጸላኢሁ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Tasabḥa samāʿta Lǝddā ba-lǝsāna sayf ṣalāʿihu... (Qǝne of wāzemā type)</title>
+                <title xml:lang="en" corresp="#t1">They wear clothes of rags...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>         
+                <p>This <term key="Qene">qǝne</term> poem, attested in <ref type="mss" corresp="Tanasee6"/>, 
+                    is of the type <foreign xml:lang="gez">wāzemā</foreign>. 
+                    It is attributed to <persName ref="PRS14503GabraMarQene">mamhǝr Gabra 
+                        Māryām of Kǝbrān</persName> (as <ref type="work" corresp="LIT7033Yelebbesu"/>
+                    and <ref type="work" corresp="LIT7036Gabreelbeta"/>).</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Wazema"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-07-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7035Tasabha" passive="Tanasee6"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7035Tasabha" passive="PRS14503GabraMarQene"/>
+                </listRelation>
+                <div type="edition">
```

Comment:
I think div for bibliography and for edition should be in parallel and not nested.